packages/forma-36-website/src/content/guidelines/contributing.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Contributing to Forma 36'
+slug: '/guidelines/contributing/'
+---
+
+Forma 36 is an open-source design system by Contentful and we welcome contributions from our community.
+If we have an idea that could help other users in this page we will teach you how to do it.
+
+## Table of contents
+
+- [Before contributing](#before-contributing)
+  - [Check if your idea is already present](#check-if-your-idea-is-already-present)
+  - [Could an existing component be extended to realize your idea?](#could-an-existing-component-be-extended-to-realize-your-idea)
+- [Contribution Process](#contribution-process)
+  - [Step 1: Open a Github issue](#step-1-open-a-github-issue)
+  - [Step 2: Proposal Review](#step-2-proposal-review)
+  - [Step 3: Open a Pull Request](#step-3-open-a-pull-request)
+  - [Step 4: Code & Design Review](#step-4-code--design-review)
+  - [Step 5: Merge the changes to the main branch](#step-5-merge-the-changes-to-the-main-branch)
+- [Bugs & Light contributions](#bugs--light-contributions)
+  - [Examples of light contributions](#examples-of-light-contributions)
+
+## Before contributing
+
+### Check if your idea is already present
+
+Before writing a proposal or any code, we ask you to go around this website or our Github repo and
+check if your idea is not yet implemented.
+It’s also a good idea to take a look on the current open proposals,
+maybe there is something similar to your idea there and you could work together with that proposer.
+
+### Could an existing component be extended to realize your idea?
+
+During your research, did you find a component that almost does what you have in mind?
+If yes, consider proposing an update or refactor of this component.
```

Comment:
```suggestion
If yes, consider proposing an update to e.g. add a new variation or a refactoring of this component.
```

Reply:
"an update to e.g."
I didn't understand this, I'm sorry
could explain what's the meaning?

Reply:
i somehow wanted to make clear what all could be considered an update. Maybe this could be clearer:
```suggestion
If yes, consider proposing an update or refactor of this component. This could be a new variation or adding a new property to the component.
```